pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -264,5 +290,53 @@ async def send_study_to_anon(self, resource_id: str) -> Any:
         )
 
         logger.debug("Successfully triggered c-store of study to anon modality: {}", resource_id)
-
         return str(response["ID"])
+
+
+class PIXLAnonOrthanc(Orthanc):
+    """Orthanc Anon connection."""
+
+    def __init__(self) -> None:
+        super().__init__(
+            url=config("ORTHANC_ANON_URL"),
+            username=config("ORTHANC_ANON_USERNAME"),
+            password=config("ORTHANC_ANON_PASSWORD"),
+            http_timeout=config("PIXL_QUERY_TIMEOUT", default=10, cast=int),
+            dicom_timeout=config("PIXL_DICOM_TRANSFER_TIMEOUT", default=240, cast=int),
+            aet=config("ORTHANC_ANON_AE_TITLE"),
+        )
+
+        self.autoroute_to_endpoint = config(
+            "ORTHANC_AUTOROUTE_ANON_TO_ENDPOINT", default=False, cast=bool
+        )
+
+    async def import_study_from_raw(self, orthanc_raw: PIXLRawOrthanc, resource_id: str) -> Any:
+        """
+        Notify Orthanc Anon of a study to pull from Raw
+
+        - Query Orthanc Raw for the study
+        - Send the StudyInstanceUID and the query ID to Orthanc Anon
+        """
+        orthanc_raw_study_info = await orthanc_raw.query_local_study(study_id=resource_id)
+        study_uid = orthanc_raw_study_info["MainDicomTags"]["StudyInstanceUID"]
```

Comment:
shouldn't these lines (and the rest of the lines in this function) be covered by the system tests? Not sure why codecov is complaining?

Reply:
oh, `pytest-cov` handles checking coverage in sub-processes, but not for processes running in Docker containers